### PR TITLE
Some notes and figures on benchmarking the SNSPowderReduction workflow

### PR DESCRIPTION
**Pull request by nvaytet**
The scripts and notes can be found in the `nvaytet` subdirectory

### CODE REVIEW

File: benchmarks/nvaytet/SANS2D/SANS2D-binning.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```
Comment (SimonHeybrock) [inline benchmarks/nvaytet/SANS2D/SANS2D-binning.sh]: Are this and the following scripts actually modified compared to my version?

File: benchmarks/nvaytet/summary.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+event splitter log: entry 0 and entry 1 are 2010-03-25T16:08:37Z and 2010-03-25T16:09:10Z.
+workspace tempsplitws3_b has 453660950 events
+event splitter log: entry 0 and entry 1 are 2010-03-25T16:08:37Z and 2010-03-25T16:09:11Z.
+workspace tempsplitws3_c has 230740000 events
+event splitter log: entry 0 and entry 1 are 2010-03-25T16:08:37Z and 2010-03-25T16:09:44Z.
+workspace tempsplitws3_unfiltered has 17288317 events
+event splitter log: entry 0 and entry 1 are 2010-03-25T16:08:37Z and 2010-03-25T16:09:10Z.
+```
+
+**Note:** the run time has significantly increased from 20s to 155s.
```
Comment (SimonHeybrock) [inline benchmarks/nvaytet/summary.md:227]: Thanks for the additional tests. While the variation is a lot, this still handles 1e9 events in approx. 100 seconds, i.e., 1e7/s. Given my baseline of a full reduction dealing with 1e6 events/s/core, 1e7/s is still not higher than 10%, i.e., your test confirm that it is not a major additional contribution.

**Review by SimonHeybrock — COMMENTED** [review]
In principle this would need tidying up a bit, but given that it just captures how some benchmarks were run I believe it is sufficient.

There are some binary files as both pdf and png, is that necessary? Is it sufficient to just keep the png, such that the markdown displays correctly?

File: benchmarks/nvaytet/SANS2D/SANS2D-binning.sh (hunk 1 above)
Comment (nvaytet) [reply benchmarks/nvaytet/SANS2D/SANS2D-binning.sh]: You are right, I had started to play with them but did not get anywhere so i reverted them back to your files. But no need to duplicate. I will remove them

**Pull request merged**